modules/enum_glue/main.py

```diff
@@ -45,6 +47,24 @@
 parser.add_argument('--dev-endpoints', required=False, default=False, action='store_true', help='Enumerate Glue development endpoints.')
 parser.add_argument('--jobs', required=False, default=False, action='store_true', help='Enumerate Glue jobs.')
 
+def fetch_glue_data(client, func, key, print, **kwargs):
+    caller = getattr(client, func)
+    try:
+        response = caller(**kwargs)
+        data = response[key]
+        while 'NextToken' in response and response['NextToken'] != '':
+            print({**kwargs, **{'NextToken': response['NextToken']}})
+            response = caller({**kwargs, **{'NextToken': response['NextToken']}})
```

Comment:
I think the 'Region' arg needs to get input here for these items (like in some of the code removed below)